### PR TITLE
Ydc/raffle queries

### PR DESCRIPTION
**Pull request by YannisDC**

### CODE REVIEW

File: Queries/AddedToLiquidationPool.psql [hunk 1]
```diff
@@ -0,0 +1,11 @@
+with pass1 as (
+  select sender_address
+  from transactions txs
+  where contract_call_contract_id = 'SP2C2YFP12AJZB4MABJBAJ55XECVS7E4PMMZ89YZR.arkadiko-liquidation-pool-v1-1'
+  and contract_call_function_name = 'stake'
+  and status = 1
+  and ( block_height > 10 /*From*/ and block_height < 10000 /*Until*/ )
```
Comment (philiphacks) [inline Queries/AddedToLiquidationPool.psql]: Q: should we still define these blocks? or?

File: Queries/AddedToLiquidationPool.psql [hunk 2]
```diff
@@ -0,0 +1,11 @@
+with pass1 as (
+  select sender_address
+  from transactions txs
+  where contract_call_contract_id = 'SP2C2YFP12AJZB4MABJBAJ55XECVS7E4PMMZ89YZR.arkadiko-liquidation-pool-v1-1'
+  and contract_call_function_name = 'stake'
+  and status = 1
```
Comment (philiphacks) [inline Queries/AddedToLiquidationPool.psql:6]: Q: what does status 1 mean?

**Review by philiphacks — CHANGES_REQUESTED** [review]
LGTM - just some questions and I guess we need to decide the weights and how many NFTs to give to each query etc

File: Queries/AddedToLiquidationPool.psql (hunk 1 above)
Comment (YannisDC) [reply Queries/AddedToLiquidationPool.psql]: They should indeed still be defined.

File: Queries/AddedToLiquidationPool.psql (hunk 2 above)
Comment (YannisDC) [reply Queries/AddedToLiquidationPool.psql:6]: 1 for successful, -1 for failed transaction

**Review by philiphacks — APPROVED** [review]
Thanks! I ran all queries now with block 68424 (around 21st of July), except for the liquidated vaults.. kept them at the current list since you already ran all of that

Gonna merge this

**Pull request merged**